# Default action type
uint8 ACTION_TYPE_DEFAULT = 0

# Base motion action types
uint8 ACTION_TYPE_MOVE_BASE_POSE = 1
uint8 ACTION_TYPE_MOVE_BASE_NAME = 2
uint8 ACTION_TYPE_DBC_POSE = 3
uint8 ACTION_TYPE_DBC_NAME = 4

# Arm motion action types
uint8 ACTION_TYPE_MOVEIT_POSE = 1
uint8 ACTION_TYPE_MOVEIT_NAME = 2
uint8 ACTION_TYPE_MOVEIT_JOINTS = 3
uint8 ACTION_TYPE_CVC_POSE = 4

uint8 type
string name

